index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>portfolio</title>
+</head>
+<body>
+  <header class="navigation">
+    <a href="#" class="logo">My Logo</a>
+    <ul class="nav">
+      <li><img src="./image/Vector.svg" alt=""></li>
+      <li><img src="./image/Vector.svg" alt=""></li>
+      <li><img src="./image/Vector.svg" alt=""></li>      
+    </ul>
```

Comment:
it is quite obvious that you have put a lot of effort into this project. However, as the previous reviewers suggested and for better semantics, it is good that your website has a nav tag to list the main navigation links of the page. Kindly fix this and your project will be ready for approval.